uint8   id
uint8	tag_id
uint8   sequence
uint64  timePollReceived
uint64  timePollAckSent
uint64  timeRangeReceived
